src/edu/csus/ecs/pc2/tools/PC2Tools.java

```diff
@@ -29,6 +30,7 @@ private static void usage() {
                 "--gen   - generate passwords to stdout, use --help --gen for usage", //
                 "--mer   - prints mail merge file to stdtout, use --help --mer for usage", //
                 "--sr   -  submit runs from CLICS JSON EF, use --help --sr for usage", //
+                "--ttsv -  convert teams.json to teams.tsv, use --help --ttsv for usage", //
```

Comment:
The argument name "ttsv" seems unintuitive; the usage comment says that it means "convert teams.json to teams.tsv".  It seems like either the argument name should be "ctsv", or the message should say "translate teams.json..."

Reply:
updated to -ctsv